app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt

```diff
@@ -126,14 +127,12 @@ class AppVersionActivityTest {
   }
 
   // TODO(#555): Create one central utility file from where we should access date format or even convert date timestamp to string from that file.
```

Comment:
Remove this TODO as you are solving it in this PR.

Reply:
Done